app/views/errors/eds_session_error.erb

```diff
@@ -0,0 +1,4 @@
+<h2 class="title title-page">An error occurred accessing the detail page you requested.</h2>
+
+<p>You may be able to access similar information this <%= link_to('alternative interface', @eds_error_link) %>.
```

Comment:
This is missing a word ("..._at_ this alternative interface"?)

Reply:
Thx. Will address with https://mitlibraries.atlassian.net/browse/DI-720